.php-cs-fixer.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+return (new PhpCsFixer\Config())
+    ->setRules([])
```

Comment:
```suggestion
    ->setRules(['@auto' => true])
```

;) [ but feel free to start smaller ]